packages/documentation/src/stories/foundation/color/color.docs.mdx

```diff
@@ -49,6 +53,13 @@ with a specific background color and the best possible contrast color for its co
 
 ## Signal colors
 
+<PostAlert type="info">
+<p slot="heading">Why does `Success-green` have white text?</p>
+<p>
+  The text of “Success-green” is white, despite the displayed contrast ratios, because with WGAC 3.0, the contrast with white is significantly greater, while it hardly differs in WCAG 2.0.
+  </p>
+</PostAlert>
+
```

Comment:
This Message can be removed, as it does not make sense to explain here, why our background utility works like this. Do not add it in the background utility story as well, because we do not show contrast values there.
And as soon as we implement the new contrast algorithm, the contrast values will change anyway.